src/psyclone/tests/dynamo0p3_stubgen_test.py

```diff
@@ -84,27 +84,29 @@ def test_kernel_stub_invalid_iteration_space():
             "'testkern_dofs_code'." in str(excinfo.value))
 
 
-def test_dynscalars_stub_err():
-    ''' Check that the DynScalarArgs constructor raises the expected
-    internal error if it encounters an unrecognised intrinsic type of
-    scalar when generating a kernel stub.
+def test_lfricscalars_stub_err():
+    ''' Check that LFRicScalarArgs._stub_declarations() raises the
+    expected internal error if it encounters an unrecognised data
+    type of a scalar argument when generating a kernel stub.
 
     '''
-    from psyclone.dynamo0p3 import DynScalarArgs
+    from psyclone.dynamo0p3 import LFRicScalarArgs
```

Comment:
Please could you move these imports up to the top.

Reply:
Moved.